### DISCUSSION TITLE
cant set up supabase authentication with sveltekit

### DISCUSSION THREAD

**Discussion by Ramimu3** [supabase/supabase#22197] [Questions]
im getting Property 'supabase' does not exist on type '{}'.js(2339)

and Module '"$env/static/public"' has no exported member 'PUBLIC_SUPABASE_URL'.ts(2305)
Module '"$env/static/public"' has no exported member 'PUBLIC_SUPABASE_ANON_KEY'.ts(2305)
even though i have them in a .env.local

**Comment by silentworks** [comment]
It's hard to say what the issue is without any code being shared. All these errors look TypeScript related, the last two errors you mention requires you to run `npm run dev` in order for the SvelteKit dev server to create the static TS files with these values. Also note that sometimes your code editor might not pick this up so you will need to restart your code editor.
